doc/man1/kdf.pod

```diff
@@ -0,0 +1,157 @@
+=pod
+
+=head1 NAME
+
+openssl-kdf,
+kdf - perform Key Derivation Function operations
+
+=head1 SYNOPSIS
+
+B<openssl kdf>
+[B<-help>]
+[B<-kdfopt>]
+[B<-keylen num>]
+[B<-out filename>]
+[B<-binary>]
+B<kdf_name>
```

Comment:
According to both Linux' [man-pages(7)](http://man7.org/linux/man-pages/man7/man-pages.7.html) and FreeBSD's [man(7)](https://www.freebsd.org/cgi/man.cgi?query=man&sektion=7), arguments should be in italics, so the above four lines should really be:

```
[B<-keylen> I<num>]
[B<-out> I<filename>]
[B<-binary>]
I<kdf_name>
```

I know that we haven't done this in most of our pages, but perhaps we should?